src/main/java/frc/robot/subsystems/implementations/arm/ArmSubsystem.java

```diff
@@ -296,7 +296,14 @@ public boolean isAtSetpoint() {
   }
 
   @Override
-  public void setLigament(MechanismLigament2d armLigament2d) {
+  public boolean isAtTargetPosition() {
```

Comment:
This looks like it does same thing as `isAtSetpoint()` function.  Use that one instead if you need to check if arm has reached target position(aka setpoint).